doc/en/components/radial-gauge.md

```diff
@@ -335,9 +335,15 @@ The scale is visual element that highlights full range of values in the gauge wh
 `sample="/gauges/radial-gauge/scale", height="320", alt="{Platform} radial gauge scale"`
 
 
-## Labels
+## Labels and Titles
 The radial gauge labels are visual elements displaying numeric values at a specified interval between values of the `MinimumValue` and `MaximumValue` properties. You can position labels by setting the `LabelExtent` property to a fraction, where 0 represents center of gauge and 1 represents outer extent of the gauge backing. Also, you can customize labels setting various styling properties such as `FontBrush` and `Font`.
 
+`TitleText` & `SubtitleText` properties are available and can be used to display both custom text for the needle. In addition, `TitleDisplaysValue` | `SubtitleDisplaysValue` will let you replace the custom text with the needle's value. So you can occupy custom text for the title but show the value via the subtitle and vice versa.
```

Comment:
Should be changed to "and can both be used to display custom text for the needle."

I would also recommend replacing the | character in the next sentence with "In addition, the `TitleDisplaysValue` and `SubtitleDisplaysValue` properties will let you...."

Reply:
done